ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -96,6 +97,17 @@ class MessagesHomeScreen extends React.Component<Props, State> {
 
   public componentDidMount() {
     this.props.refreshMessages();
+
+    // Refresh services related to messages received by the user
+    if (pot.isSome(this.props.lexicallyOrderedMessagesState)) {
```

Comment:
It should be done also when user refresh the list not only at when the component loaded.
What do you think to create a single function to reload message and refresh services?

Reply:
You're right. Now the service check is performed within the messages load (https://github.com/teamdigitale/io-app/pull/1269/commits/3f4dcb2d185013619f896ad609f9afc63f136a2b)